### ISSUE TITLE
Checkbox: "this field has an error" message not read by screen reader

### ISSUE THREAD

**Issue by jcjuett** [salesforce/design-system-react#2015]
http://design-system-react-components.herokuapp.com/?selectedKind=SLDSCheckbox&selectedStory=Checkbox%20Toggle%20%28with%20error%29&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

and all other Toggle Error examples. The following links are out of date though and http://design-system-react-components.herokuapp.com/ should be used.

https://design-system-react-co-pr-2004.herokuapp.com/?selectedKind=SLDSCheckbox&selectedStory=Doc%20site%20Snapshot%20Base&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

https://design-system-react-co-pr-2004.herokuapp.com/?selectedKind=SLDSCheckbox&selectedStory=Doc%20site%20Snapshot%20Toggle&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

https://design-system-react-co-pr-2004.herokuapp.com/?selectedKind=SLDSCheckbox&selectedStory=Doc%20site%20Toggle&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

HTML: `<div class="slds-form-element__help">This field is required</div>`

Solution: add id="error_01" after help"

Validate: Use VO to check that screen reader reads "This field has an error" when focus is on checkbox or toggle with error message

**Comment by jcjuett** [comment]
Not sure which component label to give this issue.

**Comment by interactivellama** [comment]
The invisible `input` already has an `aria-describedby` which points to the visual toggle. I’m thinking we might need two `aria-describedby` ids, so that the input is described by both.
